content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Direct Egress Traffic through Egress Gateway
```

Comment:
I do not like it - the title does not describe what Egress Gateway is used for. Note also that it is a task, and I think the title of the task could be in the imperative mood, like "Control Ingress Traffic", "Direct egress traffic..." etc. I am OK with a gerund - "Directing Egress Traffic through Egress Gateway"

Reply:
I agree that the title of a task needs to indicate an action, that's why I suggested "Using an ..." instead of just 
"Egress Gateway".

My main concern is that the title is too long. The other titles are relatively short and sweet. I don't think the title should describe what it's used for, that's in the description and the task itself. The important thing to convey is that this a task that shows how to use an egress gateway. What else, other than directing traffic through it, could it be used for?

Anyway, I'm open to other suggestions, but would highly recommend keeping it short

Reply:
OK, how about this: "Configure Egress Gateway" or "Egress Gateway Configuration"? I do not like the word use/using since it does not convey a lot of meaning.